examples/main.go

```diff
@@ -66,7 +63,7 @@ func main() {
 	r.Handle("/live/websocket", live.NewWebsocketHandler(liveConfig))
 
 	// setup http route
```

Comment:
nit: update comment to something like "// setup GoLive middleware to handle LiveView routes"